### ISSUE TITLE
JavaDoc links with full signature

### ISSUE THREAD

**Issue by httpdigest** [LWJGL/lwjgl3#120]
I noticed that the generated JavaDoc {@link ...} tags just name the method which is linked to, but omit the method parameters. This leads (at least Eclipse) to jump to the wrong target method when Ctrl+Click those links.
One example is the GL11.glDeleteTextures method with the single (IntBuffer) parameter. When clicking on the generated JavaDocs link Eclipse jumps to the very same method instead of the method with the (int, ByteBuffer) signature.
It would be nice if the links could have the full parameter types.
So the full JavaDoc of the glDeleteTextures method would be:
`Alternative version of: {@link #glDeleteTextures(int, ByteBuffer) DeleteTextures}`

**Comment by Spasi** [comment]
The generated JavaDoc works correctly, [try this](http://javadoc.lwjgl.org/org/lwjgl/opengl/GL11.html#glDeleteTextures(java.nio.IntBuffer)) for example. Works as expected in IntelliJ too.

Any NetBeans user that can try this?

**Comment by httpdigest** [comment]
Yes, thanks. It does not work in Eclipse with source navigating. JavaDoc view works fine but not in-source navigating the link elements.

**Comment by Spasi** [comment]
It's not that I have anything against Eclipse or cba to change the code (it's an one-line change). I've been working on lwjgl3 for 3 years now and I've put thought on almost every little detail. In this particular case, the intent was to make the source javadoc as human-readable as possible.

Is it worth bloating all javadoc (function signatures with 5+ parameters are not uncommon at all) to fix one action (source navigation) of one IDE?

**Comment by httpdigest** [comment]
No, it's probably not worth it if I'm being honest.
Sorry, I just want to give you the most possible feedback to "improve" LWJGL in any way I see. Of course I can only give you feedback how _I see_ things. People might of course see things differently.

The proposition is just based on how I use JavaDoc. For once, I never spent any minute reading generated HTML JavaDocs of any framework/library I used, because one generally never knows which part of the documentation is the most valuable at the current situation. This can only be found when being in the IDE and start interfacing with the API in Java. And then, I find myself navigating through the API JavaDoc sources and its JavaDoc links to figure things out. That's just how I work.
And then I found that following those link targets does not work in Eclipse, because it does not seem to follow the undocumented implementation convention of the JavaDoc tool to use the first overload method in declaration order.
That's just how that came up. :)

Did not want to disrespect your work on LWJGL.
